content/opt/run

```diff
@@ -19,6 +19,10 @@ chown -R rundeck:rundeck /var/rundeck
 chown -R rundeck:adm /var/log/rundeck
 chown -R rundeck:rundeck /var/lib/rundeck
 chown -R rundeck:rundeck /opt/rundeck-defaults
+rm -rf /tmp/rundeck
```

Comment:
anything in /tmp is not guaranteed to be there forever. There is a problem with changing uid and gid of that folder. Have a look please. When I passed a different uid and gid, rundeck was not able to write to that directory